backend/utils/seed.ts

```diff
@@ -0,0 +1,148 @@
+import { faker } from '@faker-js/faker'
```

Comment:
let's name this seed script to be a little more specific to what data in particular is being seeded here

Reply:
also i'd encourage you to be conscientious about whether we should use a random set of values every time or a consistent set of values and what the pros and cons of each approach would be

Reply:
I was thinking about this, I try doing a mix of both. I created 3 consistent users and populate them with fake contacts for now.